css-layout-classes/styles.css

```diff
@@ -1,11 +1,23 @@
-#body {
+.row {
   display: flex;
-  flex-direction: column;
   margin: 0 10%;
 }
 
 .container {
-  display: flex;
   justify-content: center;
   margin: 0 25px;
+  max-width: 100%;
```

Comment:
This should be `max-width: 800px`. Look at the first bullet point under the "Tips" section in the exercise instructions